gene_finder.py

```diff
@@ -79,8 +95,18 @@ def find_all_ORFs_oneframe(dna):
     >>> find_all_ORFs_oneframe("ATGCATGAATGTAGATAGATGTGCCC")
     ['ATGCATGAATGTAGA', 'ATGTGCCC']
     """
-    # TODO: implement this
-    pass
+    a=[]  #Create List
+    rol=0  #start our remaining letters at 0
+    for i in range(0,len(dna),3): # Look at three letters at a time
+        codon= dna[i:i+3] #assign value to codon
+        if (codon =='ATG' and rol<=0): #Restrictions for our if statement
+            orf= rest_of_ORF(dna[i:]) #assign value to orf
+            a.append(orf)             #add on to our list
+            rol=len(orf)              #add letters to our remaining letters
+        if  (rol > 0):                #ask if there are any remaining letters
+            rol = rol - 3             #if there are, take three away
+    return a                           #Display results
```

Comment:
I like how you are commenting every line to help the reader understand what is going on! It is generally good practice to write readable code. But sometimes, especially when the code is simple and trivial, we don't need to leave comments for "every line" if it's taking too much time.